wagtail_newsletter/wagtail_hooks.py

```diff
@@ -84,18 +90,32 @@ def redirect_to_campaign_page(request, page: Page):
         )
         return
 
-    if action == "save_campaign":
-        actions.save_campaign(request, page)
+    with LogContext(user=request.user):
+        if action == "save_campaign":
+            actions.save_campaign(request, page)
 
-    if action == "send_test_email":
-        actions.send_test_email(request, page)
+        if action == "send_test_email":
+            actions.send_test_email(request, page)
 
-    if action == "send_campaign":
-        actions.send_campaign(request, page)
+        if action == "send_campaign":
+            actions.send_campaign(request, page)
 
 
 @hooks.register("after_copy_page")  # type: ignore
 def clear_campaign_after_copy(request, page, new_page):
     if isinstance(new_page, NewsletterPageMixin) and new_page.newsletter_campaign:
         new_page.newsletter_campaign = ""
         new_page.save(update_fields=["newsletter_campaign"])
+
+
+@hooks.register("register_log_actions")  # type: ignore
+def register_log_actions(actions):
+    actions.register_action(
+        "wagtail_newsletter.save_campaign", "Save campaign", "Campaign saved"
+    )
+    actions.register_action(
+        "wagtail_newsletter.send_test_email", "Send test email", "Test email sent"
+    )
+    actions.register_action(
+        "wagtail_newsletter.send_campaign", "Send campaign", "Campaign sent"
+    )
```

Comment:
I haven't marked any strings as translatable in the package. I'd argue it would be better to _not_ have any marked as translatable until we're ready to set up and maintain translations on this package.

Reply:
OK, I saw that the `verbose_name` and `verbose_name_plural` for `NewsletterLogEntry` are using `gettext_lazy`, so I asked

Reply:
Whoops, I copy/pasted that code without realising that it's using translation. I'll remove it. Thanks!

Reply:
I've removed the code; not relevant any more.